bin/reth-benchmark/Cargo.toml

```diff
@@ -0,0 +1,101 @@
+[package]
```

Comment:
any reason why we don't call this `reth bench`?

Reply:
no reason, should be an ez rename